docs/user/masternodes/setup-testnet.rst

```diff
@@ -756,8 +763,14 @@ node is working properly. Install Sentinel as follows::
   bin/python bin/sentinel.py
 
 You will see a message reading **dashd not synced with network! Awaiting
```

Comment:
The preceding instructions don't result in a working Sentinel for me (even if I switch the sentinel.conf from mainnet -> testnet). Can't wait for sentinel to go away! Either the RPC credentials need to be passed by [env variables](https://github.com/dashpay/sentinel/tree/master#configuration) or it needs to be pointed to dash.conf.

```
deprecation warning: JSONRPC credentials should now be set using environment variables. Using dash.conf will be deprecated in the near future.
[Errno 111] Connection refused
Cannot connect to dashd. Please ensure dashd is running and the JSONRPC port is open to Sentinel.
```

Reply:
This works for me, I get `dashd not synced with network! Awaiting full sync before running Sentinel.` after following the instructions. Sentinel should try to read `dash.conf` from `~/.dashcore/dash.conf` by default, and since this file already exists and contains `testnet=1` no further configuration is necessary. Can you check you have this file and are running both core and sentinel as the same user?